db/schema.rb

```diff
@@ -293,6 +299,8 @@
     t.datetime "active_status_updated_at"
     t.string "banner_style"
     t.text "body_markdown"
+    t.integer "broadcastable_id"
+    t.string "broadcastable_type"
```

Comment:
I think we probably want to add an index on broadcast for the `broadcastable_id` and `broadcastable_type`. Since polymorphic associations rely on querying both of these together to figure out _which_ type of broadcast we're dealing with, we probably want a combined index here, something like this?
```ruby
add_index :broadcasts, [ : broadcastable_type, : broadcastable_id]
```

To learn more about indexes + polymorphic associations, check out [this](https://mauricio.github.io/2008/09/27/handling-database-indexes-for-rails-polymorphic-associations.html) blog post!

Reply:
Also just want to note that we have a `index_broadcasts_on_title_and_type_of` currently on the `broadcasts` table; we'll want to explicitly remove that whenever we remove the `type_of` column :)

Reply:
Question: can the combo "type" and "id" allow more than one item? Otherwise it should be a unique index as well, by adding `unique: true` to the parameters

Reply:
Making these changes now! Thank you both for pointing all of this out and linking to the post @vaidehijoshi @rhymes. 😄